README.md

```diff
@@ -55,19 +56,21 @@ Please note, that this is a **Partner Supported** module, which means that techn
 ### Compatibility - supported Sensu versions
 
 If not explicitly stated it should always support the latest Sensu release.
-Beginning with v4.0.0 this module will only support Sensu Go 5.16+.
+Beginning with v5.0.0 this module will only support Sensu Go 6.0+.
 Please log an issue if you identify any incompatibilities.
 
 | Sensu Go Version| Recommended Puppet Module Version   |
 | --------------- | ----------------------------------- |
 | 5.0 - 5.15      | latest v3                           |
 | 5.16+           | latest v4                           |
+| 6.0+            | latest v5                           |
 
 ### Upgrade note
 
 Sensu Go 5.x is a rewrite of Sensu and no longer depends on redis and rabbitmq.
 Version 3 of this module supports Sensu Go 5.0.0 to 5.15.x.
 Version 4 of this module supports Sensu Go 5.16+.
+Version 5 of this module supports Sensu Go 6.0+.
```

Comment:
We should change to use notation like this
```
>= 5.0.0 < 5.16.0
>= 5.16.0 < 6.0.0
>= 6.0.0 < 7.0.0
```

Reply:
Changed to use that notation